Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/server/DataDonationAnalyticsApiV1.kt

```diff
@@ -0,0 +1,13 @@
+package de.rki.coronawarnapp.datadonation.analytics.server
+
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.PpaDataRequestAndroid
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface DataDonationAnalyticsApiV1 {
+    @POST("/version/v1/android/dat")
+    suspend fun submitAndroidAnalytics(
```

Comment:
Can I have a test for this stuff too? :cat2: 

Should be straight forward, check the other API tests: Making sure serialization does what it should, bad data leads to an error, and the API path is what we expect.